lib/rack/attack.rb

```diff
@@ -158,19 +76,16 @@ def call(env)
       env['PATH_INFO'] = PathNormalizer.normalize_path(env['PATH_INFO'])
       request = Rack::Attack::Request.new(env)
 
-      if safelisted?(request)
+      if @config.safelisted?(request)
```

Comment:
Can we please use the reader instead of the instance variable so that it's easier to detect typos?